### PR TITLE
Clean reports

### PR DESCRIPTION
**Pull request by Parastoo-S**
Created a new pull request for cleaned up the Reports commits.

### CODE REVIEW

File: app/controllers/report_generator_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class ReportGeneratorController < AdminAreaController
+#Not sure if I should keep this controller
```
Comment (tahnok) [inline app/controllers/report_generator_controller.rb]: Probably not necessary

Comment (Parastoo-S) [inline app/controllers/report_generator_controller.rb]: sure. will remove it

**Comment by Parastoo-S** [comment]
you're right. I was just playing around with the emails. I will add them to a new PR

File: config/routes.rb [hunk 2]
```diff
@@ -96,11 +107,19 @@
     end
   end
 
+  namespace :report_generator do
```
Comment (tahnok) [inline config/routes.rb]: Is this left over?

Comment (Parastoo-S) [inline config/routes.rb]: cleaned.

File: test/fixtures/users.yml [hunk 3]
```diff
@@ -1,37 +1,75 @@
...
+  password: $2a$10$CdKIia2HsjiFqYWtmDbV2ed4SY2Yn0oU.JdcIzPq09FSg71yZxIzq
+  url: null
+  location: makerspace
+  gender: female
+  faculty: engineering
+  use: null
+  description: null
+  role: staff
+  identity: faculty_member
+  created_at: 2017-06-21T13:25:01-04:00
```
Comment (tahnok) [inline test/fixtures/users.yml]: In order for this not to fail in the future once we are not a month away from July, you should probably use the templating system that fixtures have and make this `created_at: <%= 1.month.ago %>` or something like that

File: config/routes.rb [hunk 4]
```diff
@@ -75,6 +75,16 @@
   namespace :admin do
     get 'index', path: '/'
 
+    resources :report_generator do
```
Comment (tahnok) [inline config/routes.rb]: I'm not sure this `resources : report_generator` is the way you want to do this. Look at the output of rake routes:

```
report1_admin_report_generator_index GET    /admin/report_generator/report1(.:format)                      admin/report_generator#report1
report2_admin_report_generator_index GET    /admin/report_generator/report2(.:format)                      admin/report_generator#report2
report3_admin_report_generator_index GET    /admin/report_generator/report3(.:format)                      admin/report_generator#report3
                                     POST   /admin/report_generator/report1(.:format)                      admin/report_generator#report1
                                     POST   /admin/report_generator/report2(.:format)                      admin/report_generator#report2
                                     POST   /admin/report_generator/report3(.:format)                      admin/report_generator#report3
        admin_report_generator_index GET    /admin/report_generator(.:format)                              admin/report_generator#index
                                     POST   /admin/report_generator(.:format)                              admin/report_generator#create
          new_admin_report_generator GET    /admin/report_generator/new(.:format)                          admin/report_generator#new
         edit_admin_report_generator GET    /admin/report_generator/:id/edit(.:format)                     admin/report_generator#edit
              admin_report_generator GET    /admin/report_generator/:id(.:format)                          admin/report_generator#show
                                     PATCH  /admin/report_generator/:id(.:format)                          admin/report_generator#update
                                     PUT    /admin/report_generator/:id(.:format)                          admin/report_generator#update
                                     DELETE /admin/report_generator/:id(.:format)                          admin/report_generator#destroy
                  search_admin
```

There's a lot of things in there from the 'classic' 7 routes. You'll have to do some search on how to do this properly. Maybe `namespace :report_generator` or you can exclude actions

File: test/fixtures/users.yml (hunk 3 above)
Comment (Parastoo-S) [inline test/fixtures/users.yml]: ooh true. will do that. thanks

File: config/routes.rb (hunk 4 above)
Comment (Parastoo-S) [inline config/routes.rb]: okay I will look into it

File: config/routes.rb [hunk 5]
```diff
@@ -75,14 +75,11 @@
   namespace :admin do
     get 'index', path: '/'
 
-    resources :report_generator do
+    resources :report_generator, only: [:index] do
```
Comment (Parastoo-S) [inline config/routes.rb]: @tahnok Does this look better?

**Pull request merged**